learn/guides/organizing-ballerina-code/package-layout.md

```diff
@@ -117,6 +117,51 @@ codeCoverage = true
 cloud = "k8s"
 ```
 
+### Packaging Java Libraries
+
+When you compile a Ballerina package with `bal build`, the compiler creates an executable JAR file. However, if the package does not contain an entry point, it will produce a non-executable JAR file (a library package), which can be used in another package/program. In both cases, the Ballerina compiler produces self-contained archives. There are situations in which you need to package JAR files with these archives.
+
+You are free to store the JAR files anywhere in your file system. As a best practice, maintain Java libraries inside the package.
+The platform-specific library information needs to be specified in the `Ballerina.toml` file. Java libraries are considered as platform-specific libraries.
+Here, is how you can specify a JAR file dependency in the `Ballerina.toml` file.
+
+```toml
+[[platform.java11.dependency]]
+# Absolute or relative path to the JAR file
+path = "<path-to-jar-file-1>"
+# An optional comma-separated list of Ballerina module names (to restrict the usage of this JAR)
```

Comment:
```suggestion
# An optional comma-separated list of Ballerina module names (to restrict the usage of this JAR)
```
```suggestion
# An optional comma-separated list of Ballerina module names (to restrict the usage of this JAR).
```

Reply:
Please add the full stop in all code comments in all occurrences.

Reply:
Done.